extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,11 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery-mask {
+  display: none !important;
+  z-index: 2147483646 !important;
+}
+
+.-amp-lightbox-viewer[gallery-view] .-amp-lightbox-viewer-gallery-mask {
+  display: block !important;
+}
+
+.-amp-lightbox-viewer-gallery-grid {
+  margin-top: 50px;
+  height: 100vh;
+  width: 100vw;
+  display: flex 0 0 auto;
+}
+
+.-amp-lightbox-viewer-gallery-thumbnail {
```

Comment:
let's rewrite this as :

```
.-amp-lightbox-viewer-gallery-thumbnail {
  display: inline-block;
  width: calc(100vw/3);
  height: calc(100vw/3);
  padding: 5px;
  box-sizing: border-box;
  overflow: hidden;
}
```

also add a 5px padding to `.-amp-lightbox-viewer-gallery` so the gutter side is the same between images and also between the edges and the images